pkg/convert/tf.go

```diff
@@ -1703,10 +1715,34 @@ func rewriteTraversal(
 	return hclwrite.TokensForTraversal(newTraversal)
 }
 
+func isModuleRef(expr *hclsyntax.ScopeTraversalExpr) (string, bool) {
+	root := expr.Traversal.RootName()
+	if root != "module" {
+		return "", false
+	}
+	if attr, ok := expr.Traversal[1].(hcl.TraverseAttr); ok {
```

Comment:
Is [1] guaranteed to exist or needs a bounds check?

Reply:
Yes, traversal expressions in hcl always start with the type of the thing we are traversing e.g. `module`, `local`, `resource` etc then the name reference. For example `module.example_module` is a traversal that accesses the `example_module` so it does not require a bounds check